AsyncDisplayKit/ASTextNode.mm

```diff
@@ -250,9 +248,28 @@ - (ASTextKitAttributes)_rendererAttributes
   };
```

Comment:
We only call the `_rendererAttributes` method within the `_rendererWithBounds:` method where we already hold the lock that's the reason I didn't add locking to it for now. Would you recommend to add locking to `_rendererAttributes` too?